dev-libs/boehm_gc/boehm_gc-7.6.4.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="The Boehm-Demers-Weiser conservative garbage collector"
+DESCRIPTION="The Boehm-Demers-Weiser conservative garbage collector can be \
+used as a garbage collecting replacement for C malloc or C++ new. It allows \
+you to allocate memory basically as you normally would, without explicitly \
+deallocating memory that is no longer useful. The collector automatically \
+recycles memory when it determines that it can no longer be otherwise \
+accessed."
+HOMEPAGE="http://www.hboehm.info/gc/"
+COPYRIGHT="1988, 1989 Hans-J. Boehm, Alan J. Demers
+	1991-1996 Xerox Corporation
+	1996-1999 Silicon Graphics
+	1999-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="Boehm"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/bdwgc/releases/download/v$portVersion/gc-$portVersion.tar.gz"
```

Comment:
Shorter link

```diff
-SOURCE_URI="https://github.com/ivmai/bdwgc/releases/download/v$portVersion/gc-$portVersion.tar.gz"
+SOURCE_URI="http://www.hboehm.info/gc/gc_source/gc-$portVersion.tar.gz"

Reply:
It's shorter, but it's not https and the first URI where the archive was available from is on GitHub, as we can see with the timestamps:
```
TZ=UTC ls -go --fu github/gc-7.6.4.tar.gz shorter-uri/gc-7.6.4.tar.gz
-rw-r--r-- 1 1120784 2018-01-26 13:19:23.000000000 +0000 github/gc-7.6.4.tar.gz
-rw-r--r-- 1 1120784 2018-01-29 03:06:48.000000000 +0000 shorter-uri/gc-7.6.4.tar.gz
```
Moreover, on GutHub we are sure that the archives won't be removed, whereas on bboehm.info old tarballs might be removed.